pkg/testrunner/template/types.go

```diff
@@ -38,6 +38,7 @@ type TestrunParameters struct {
 	FloatingPoolName        string
 	LoadBalancerProvider    string
 	ComponentDescriptorPath string
+	GardenerVersion         string
```

Comment:
nit: add a new line to separate this value from the other normal parameters

Reply:
done